service/tablet_allocator.cc

```diff
@@ -819,8 +997,12 @@ class tablet_allocator_impl : public tablet_allocator::impl
         _stopped = true;
     }
 
-    future<migration_plan> balance_tablets(token_metadata_ptr tm) {
+    future<migration_plan> balance_tablets(token_metadata_ptr tm, std::optional<locator::load_stats> table_load_stats) {
```

Comment:
Does this draft contain the code to do the actually splitting of the tablets? I mean the code to update the number of tablets and the replicas for each tablets? Could you point me the code if any?

Reply:
not in this RFC, although I already have it implemented. the next version will contain it. thanks for ur review.

Reply:
Is it in the new version https://github.com/scylladb/scylladb/pull/16580#issuecomment-1895012331?